ci/checksizes.sh

```diff
@@ -45,10 +45,10 @@ case $FULLVER in
     REVERSI_PREOPT_GZ_EXPECTEDSIZE=71000
     REVERSI_OPT_GZ_EXPECTEDSIZE=32000
     ;;
-  2.12.0-M3)
-    REVERSI_PREOPT_EXPECTEDSIZE=530000
-    REVERSI_OPT_EXPECTEDSIZE=123000
-    REVERSI_PREOPT_GZ_EXPECTEDSIZE=72000
+  2.12.0-M4)
+    REVERSI_PREOPT_EXPECTEDSIZE=593000
```

Comment:
The `REVERSI_PREOPT_EXPECTEDSIZE` increased by 12%. Is there any known reason for this?